query/src/catalogs/impls/catalog/metastore_catalog.rs

```diff
@@ -218,4 +218,17 @@ impl Catalog for MetaStoreCatalog {
 
         Ok(tbl)
     }
+
+    async fn exists_database(&self, db_name: &str) -> Result<bool> {
+        match self.get_database(db_name).await {
+            Ok(_) => Ok(true),
+            Err(err) => {
+                if err.code() == ErrorCode::UnknownDatabase("").code() {
```

Comment:
ErrorCode::UnknownDatabase("").code() -> ErrorCode::UnknownDatabaseCode() is neat

Reply:
resolved